portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeErcFromAssetLibraryForStructuredContents.testcase

```diff
@@ -59,11 +59,38 @@ definition {
 				title = "WC WebContent Title");
 		}
 
-		task ("Then structured Content is being createdAnd Then I can see the custom erc in the body response") {
+		task ("Then structured content is being created and I can see the custom erc in the body response") {
```

Comment:
the **And** word is reserved for cucumber syntaxis, can you modify the scenario so it does not appear?

Reply:
Updated in forced push.